process-engine/logging_api
# ♻️ Merge Logging and Metric APIs

## Changes

1. Add `measuredAt`, `tokenPayload` and `error` to the log files
2. Import `MetricMeasurementPoint` enum
3. Add support for versioning to the logfile repository
4. Add error serialization/deserialization to the logfile repository
5. Append the new properties to the end of each log entry to ensure backwards compatibility

**Note:** The Logging API now effectively takes over for the Metrics API; which will be marked as deprecated, as soon as everything is deployed.

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/429

PR: #2

## How to test the changes

- Run a few ProcessInstances
- Examine the produced logs
- Notice how they each now contain a token payload, a measuerment point and optionally an error
